src/webcore/macros.rs

```diff
@@ -654,6 +661,7 @@ macro_rules! reference_boilerplate {
     };
 }
 
+#[macro_export]
```

Comment:
Hmm... I assume you're exporting these to allow the context to be implemented externally? Hmmm.... Could we keep those private? I would actually be fine having the `RenderingContext`s implementations inside `stdweb` for now. (They're JS APIs after all!)

Reply:
Hm, OK, but I think these macros should be exposed anyway - it's not uncommon to want to wrap new types of references.

Reply:
I'm not against exposing something like this macro in the future, but I'd rather keep it private for now. Unlike the `js!` macro this macro wasn't designed to be used outside of `stdweb`, so I will likely want to export something *slightly* different to the outside.